utilities/tests/tree-test.metta

```diff
@@ -5,7 +5,10 @@
 ! (import! &self metta-moses:utilities:tree)
 ! (import! &self metta-moses:utilities:nodeId)
 ! (import! &self metta-moses:utilities:general-helpers)
+! (import! &self metta-moses:reduct:enf)
 
+;; Helper function for clean tree
+(= (REDUCE $expr) (reduce $expr))
```

Comment:
Why do you need to define another helper function `REDUCE` when you can directly use the `reduce` itself ?